src/content/news.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import NewsEntry from './news_entry'
+import PrevNextButton from './prev_next'
+import NewsContent from './news_content'
+
+export default class News extends React.Component {
+    constructor(props) {
+        super();
+        this.state = {
+            // page: props.page,
+            // page: [1, 0, 0],
+            default: true,
+            index: 0,
+            news: [{
+                src: "1",
+                alt: "1",
+                text: "Text 1"
+            },
+            {
+                src: "2",
+                alt: "2",
+                text: "Text 2"
```

Comment:
News wouldn't consist of one word. Please use some word generator. F.e.: https://lipsum.com/